domains/asaad.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "asad9200",
+        "email": "muhammadasadullah920@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
invalid Domain Structure, please read -> https://docs.is-a.dev/domain-structure/